defaults/aws_config.yaml

```diff
@@ -8,22 +8,22 @@ instance_type_loader: 'c5.xlarge'
 instance_type_monitor: 't3.large'
 regions_data:
   us-east-1: # US East (N. Virginia)
-    ami_id_loader: 'ami-04d4eb45311c3d153' # Loader dedicated AMI v17
+    ami_id_loader: 'ami-04ba971b593b243c5' # Loader dedicated AMI v18
```

Comment:
perhaps better to consider adding this change in a separate PR since doesn't look related to new nodetool command enhancement.

Reply:
yes, I'm waiting for https://github.com/scylladb/scylla-cluster-tests/pull/4587 to be merged. This one is needed for testing.

Reply:
rebased